great_expectations/self_check/util.py

```diff
@@ -1587,6 +1585,12 @@ def generate_expectation_tests(
                     c, d["data"], d["schemas"]
                 )
             except Exception as e:
+                # Adding these print staments for build_gallery.py's console output
+                print("\n\n[[ Problem calling get_test_validator_with_data ]]")
+                print(f"expectation_type -> {expectation_type}")
+                print(f"c -> {c}\ne -> {e}")
+                print(f"d['data'] -> {d.get('data')}")
+                print(f"d['schemas'] -> {d.get('schemas')}")
```

Comment:
❤️ 

This surfaces Spark mixed type & invalid column header errors that otherwise fail silently.

These failures cause the tests to not be collected or counted as failing tests; we may want to address that at some point in the future but this is miles ahead of not surfacing those issues at all.

Reply:
Yea it will be great to see all these kinds of things resolved in the future